src/Controls/src/Core/HandlerImpl/Label/Label.Android.cs

```diff
@@ -33,32 +33,21 @@ private protected override void OnHandlerChangedCore()
 			}
 		}
 
-		public static void MapTextType(LabelHandler handler, Label label) => MapText((ILabelHandler)handler, label);
+		public static void MapTextType(LabelHandler handler, Label label) => MapTextType((ILabelHandler)handler, label);
 		public static void MapText(LabelHandler handler, Label label) => MapText((ILabelHandler)handler, label);
 		public static void MapLineBreakMode(LabelHandler handler, Label label) => MapLineBreakMode((ILabelHandler)handler, label);
 
 
 		public static void MapTextType(ILabelHandler handler, Label label)
 		{
-			Platform.TextViewExtensions.UpdateText(handler.PlatformView, label);
+			handler.UpdateValue(nameof(ILabel.Text));
 		}
 
 		public static void MapText(ILabelHandler handler, Label label)
 		{
 			Platform.TextViewExtensions.UpdateText(handler.PlatformView, label);
 		}
 
-		// TODO: NET8 make this public
-		internal static void MapTextColor(ILabelHandler handler, Label label)
```

Comment:
No longer needed as the new way of `Platform.TextViewExtensions.UpdateText()` correctly handles span colors. There is no difference with either HTML of formatted text.